lib/ui/src/modules/ui/components/layout/index.js

```diff
@@ -130,7 +130,7 @@ class Layout extends React.Component {
   constructor(props) {
     super(props);
 
-    this.layerSizes = getSavedSizes(defaultSizes);
+    this.layerSizes = getSavedSizes(defaultSizes, props.isMobileDevice);
```

Comment:
Looks like this second argument isn't used anymore

Reply:
yeah, you are right ) please advise how could I clean it up?